### PR TITLE
[NUM-13] Created a navigation bar

### PR DESCRIPTION
**Pull request by ErickS09**
## Description

A component was created for the navigation bar in the same way a subcomponent was created for each option (start, activities, etc). This component paints the bottom border pink if it is active.

## Trello

Include NUM-13

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Review by nthchild1 — APPROVED** [review]

File: src/components/TopNavbar/NavbarOption.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    return (
+        <div className={` ${styles[type]}`}>
+           {nav} 
+        </div>
+    )
+}
+export default NavbarOption
+NavbarOption.defaultProps = {
+    type: 'primary',
+    type: 'secondary'
```
Comment (nthchild1) [inline src/components/TopNavbar/NavbarOption.js:15]: Aquí no se debería definir como default prop a type dos veces.

File: src/components/TopNavbar/TopNavbar.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import NavbarOption from './NavbarOption'
+
+function TopNavbar() {
+    return (
+        <div>
```
Comment (Georgge) [inline src/components/TopNavbar/TopNavbar.js:5]: Tener en concideración: esto deberia ser un "ul" como se habia planteado

File: src/components/TopNavbar/NavbarOption.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import styles from './topnavbar.module.css'
+import {Link} from 'react-router-dom'
+
+function NavbarOption({ nav, type }) {
+
+    return (
+        <div className={` ${styles[type]}`}>
```
Comment (Georgge) [inline src/components/TopNavbar/NavbarOption.js:7]: Esto deberia ser un "li"

Comment (Georgge) [reply src/components/TopNavbar/NavbarOption.js:7]: Debe de tener una propiedad que se llame "active" que haga que el borde permanezca activo (borde tosa) cuando esta propiedad es ```true```

**Review by Georgge — COMMENTED** [review]
@ErickS09 checa los cometarios, lo de "ul" y "li" solo tenlo en cuenta no es necesario cambiarlo, lo demas si arreglalo porfa.

**Pull request merged**